NetKit/Classes/Core/Request/RequestConfiguration.swift

```diff
@@ -10,17 +10,13 @@ import UIKit
 
 
 var defaultTimeOut = 3.0
-var defaultRetryCount = 3
+var defaultRetryCount = 4
 
-public class RequestConfiguration {
+@objc public class RequestConfiguration: NSObject {
 
     public var readTimeOut: Double = defaultTimeOut
     public var writeTimeOut: Double = defaultTimeOut
     public var connectTimeOut: Double = defaultTimeOut
```

Comment:
No need for `@objc` here?

Reply:
Theses are not used at all, so I didn't want to just add it.
But I kept them cause I saw we have them in Android.